hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/v2/components/tables/containersTable.tsx

```diff
@@ -0,0 +1,258 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState, useRef } from 'react';
+import moment from 'moment';
+import filesize from 'filesize';
+import Table, {
+  ColumnProps,
+  ColumnsType,
+  TablePaginationConfig
+} from 'antd/es/table';
+import {
+  Container, ContainerKeysResponse, ContainerReplicas,
+  ContainerTableProps,
+  ExpandedRow, ExpandedRowState, KeyResponse
+} from '@/v2/types/container.types';
+import { getFormattedTime } from '@/v2/utils/momentUtils';
+import { NodeIndexOutlined } from '@ant-design/icons';
+import { Popover } from 'antd';
+import { AxiosGetHelper } from '@/utils/axiosRequestHelper';
+import { showDataFetchError } from '@/utils/common';
+import { AxiosError } from 'axios';
+
+const size = filesize.partial({ standard: 'iec' });
+
+export const COLUMNS: ColumnsType<Container> = [
+  {
+    title: 'Container ID',
+    dataIndex: 'containerID',
+    key: 'containerID',
+    sorter: (a: Container, b: Container) => a.containerID - b.containerID
+  },
+  {
+    title: 'No. of Keys',
+    dataIndex: 'keys',
+    key: 'keys',
+    sorter: (a: Container, b: Container) => a.keys - b.keys
+  },
+  {
+    title: 'Actual/Expected Replica(s)',
+    dataIndex: 'expectedReplicaCount',
+    key: 'expectedReplicaCount',
+    render: (expectedReplicaCount: number, record: Container) => {
+      const actualReplicaCount = record.actualReplicaCount;
+      return (
+        <span>
+          {actualReplicaCount} / {expectedReplicaCount}
+        </span>
+      );
+    }
+  },
+  {
+    title: 'Datanodes',
+    dataIndex: 'replicas',
+    key: 'replicas',
+    render: (replicas: ContainerReplicas[]) => {
+      const renderDatanodes = (replicas: ContainerReplicas[]) => {
+        return replicas?.map((replica: any, idx: number) => (
+          <div key={idx} className='datanode-container-v2'>
+            <NodeIndexOutlined /> {replica.datanodeHost}
+          </div>
+        ))
+      }
+
+      return (
+        <Popover
+          content={renderDatanodes(replicas)}
+          title='Datanodes'
+          placement='bottomRight'
+          trigger='hover'>
+          <strong>{replicas.length}</strong> datanodes
+        </Popover>
+      )
+    }
+  },
+  {
+    title: 'Pipeline ID',
+    dataIndex: 'pipelineID',
+    key: 'pipelineID'
+  },
+  {
+    title: 'Unhealthy Since',
+    dataIndex: 'unhealthySince',
+    key: 'unhealthySince',
+    render: (unhealthySince: number) => getFormattedTime(unhealthySince, 'lll'),
+    sorter: (a: Container, b: Container) => a.unhealthySince - b.unhealthySince
+  }
+];
+
+const KEY_TABLE_COLUMNS: ColumnsType<KeyResponse> = [
+  {
+    title: 'Volume',
+    dataIndex: 'Volume',
+    key: 'Volume'
+  },
+  {
+    title: 'Bucket',
+    dataIndex: 'Bucket',
+    key: 'Bucket'
+  },
+  {
+    title: 'Key',
```

Comment:
To enhance the system, we can add a new column called "`Complete Path`," which will utilize the completePath parameter from the container endpoint's JSON response. This response is retrieved when we attempt to fetch information about the keys associated with the containers.
<img width="1297" alt="image" src="https://github.com/user-attachments/assets/39b7f47c-ca40-4270-9bbd-ee2638386f57">

Reply:
Added the field as a part of [3f9953c](https://github.com/apache/ozone/pull/7267/commits/3f9953c0fbbd2175ee83e8f0b4927e45e9c10ac1)